src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -1058,6 +1058,26 @@ abstract class Test
                 Diagnostic(ErrorCode.WRN_UnreferencedFieldAssg, "e").WithArguments("Test.e").WithLocation(6, 29));
         }
 
+        [Fact]
+        public void ExternEventInitializer()
```

Comment:
>ExternEventInitializer [](start = 20, length = 22)

Consider moving this test to src\Compilers\CSharp\Test\Symbol\Symbols\Source\EventTests.cs